srcpkgs/anydesk/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'anydesk'
+pkgname=anydesk
+version=4.0.1
+revision=1
+archs="i686 x86_64"
+wrksrc=${pkgname}-${version}
+short_desc="Fast remote desktop application"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="Free for personal use"
```

Comment:
Fix license to use an SPDX identifier.

Reply:
Not a Free Software license.

Reply:
Then use `custom`. You're already installing a `vlicense` here.

Reply:
Will do. The vlicense is for the third party sw that is bundled in the binary.

Reply:
Sorry I meant `license="Proprietary"`. Not `custom`.

Reply:
Fixed.